lib/AST/ASTImporter.cpp

```diff
@@ -1421,13 +1423,22 @@ bool ASTNodeImporter::ImportDefinition(VarDecl *From, VarDecl *To,
   if (To->getAnyInitializer())
     return false;
 
```

Comment:
Really this function does not ensure the definition, only the initializer? (If "definition" is not the same as initializer.)

Reply:
Yes, the definition may be independent from the initializer.
Perhaps a better name would be just `ImportInitializer`.

Reply:
Renamed to `ImportInitializer`.